src/util/validatePeer.ts

```diff
@@ -0,0 +1,13 @@
+const url = require('url')
+const IGNORE_LIST = ['localhost', '0.0.0.0', 'local.codius.org']
+
+export function validatePeer (peer: string): boolean {
+  let hostName = url.parse(peer)
+  hostName = hostName.hostname
+
+  if (hostName && hostName.slice(0.3) === '127' || IGNORE_LIST.indexOf(hostName) > -1) {
```

Comment:
I don't think this works. Somebody could still create their own version of `local.codius.org`, advertise that and it would still confuse codiusds around the world.